webapp/loggedin/tableViews/actions.js

```diff
@@ -1,22 +1,24 @@
 import axios from 'axios'
 
 import * as TableViewsState from './tableViewsState'
+import * as SurveyState from '../../survey/surveyState'
 
 export const tableViewsListUpdate = 'tableViews/list/update'
 
-const getItems = (moduleApiUri, offset, limit) => axios.get(
+const getItems = (moduleApiUri, cycle, offset, limit) => axios.get(
```

Comment:
why cycle?? let's pass custom params from the component. you also need it to count, otherwise you count all records!

Reply:
I'll add a "restParams" property to the TableView